uiowa/uiowa
# UIPress authors migration

Resolves #4663 





# How to test


Sync `uipress` in DevDesktop (grad container)
`git checkout uipress-authors-migrate && git pull`
`blt ds --site=uipress.uiowa.edu`
`drush @uipress.local en uipress_migrate -y`
`drush @uipress.local uli /admin/config/sitenow/sitenow-migrate`
Complete the form using the help text
`drush @uipress.local ms` <- check that migrations register
`drush @uipress.local mim --all`
See that migration completes, and that 'low dimension image' reporting works
Spot check some authors
